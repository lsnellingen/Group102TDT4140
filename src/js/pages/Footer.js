// Libs
import React, { Component } from 'react';

class Footer extends Component {
  render() {
    return (
      <footer className="footer">
      <div className="container">
        <div className="row">
          <div className="col-xs-8 col-xs-offset-1">
            <span className="text-muted">Here we will put information regarding our site</span>
          </div>
        </div>
      </div>
    </footer>
    );
  }
}

export default Footer;
