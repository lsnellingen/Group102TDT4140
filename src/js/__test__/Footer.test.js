import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import Footer from '../pages/Footer';

describe('Footer', () => {
  var ReactTestUtils = require('react-addons-test-utils') // ES5 with npm

    test('renders without crashing', () => {
      /*const renderer = ReactTestUtils.createRenderer();*/

      const tree = renderer.create(
        <Footer/>).toJSON();


      expect(tree).toMatchSnapshot();




    });

});
