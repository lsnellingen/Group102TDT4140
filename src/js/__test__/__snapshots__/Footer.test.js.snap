// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders without crashing 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-xs-8 col-xs-offset-1"
      >
        <span
          className="text-muted"
        >
          Here we will put information regarding our site
        </span>
      </div>
    </div>
  </div>
</footer>
`;
